nncf/onnx/quantization/ignored_patterns.py

```diff
@@ -88,3 +89,69 @@ def create_multihead_attention_output() -> GraphPattern:
     _add_softmax_matmul(pattern)
     _add_softmax_reshape_matmul(pattern)
     return pattern
+
+
+@ONNX_IGNORED_PATTERNS.register(IgnoredPatternNames.SE_BLOCK)
+def create_se_block() -> GraphPattern:
+    #       NON_PATTERN_NODE--------
+    #    (PATTERN_NODE_TO_EXCLUDE) |
+    #              |               |
+    #         REDUCE_MEAN          |
+    #              |               |
+    #         CONVOLUTION          |
+    #              |               |
+    #          ACTIVATION          |
+    #              |               |
+    #         CONVOLUTION          |
+    #              |               |
+    #     SIGMOID||HARDSIGMOID     |
+    #              |               |
+    #              |               |
+    #             MUL---------------
+    #   (PATTERN_NODE_TO_EXCLUDE)
+    pattern = GraphPattern()
+    non_pattern_node = pattern.add_node(label="NON_PATTERN_NODE", type=GraphPattern.NON_PATTERN_NODE_TYPE)
+    reduce_mean_node = pattern.add_node(
+        **{
+            GraphPattern.LABEL_ATTR: "REDUCE_MEAN",
+            GraphPattern.METATYPE_ATTR: om.ONNXReduceMeanMetatype,
+            GraphPattern.PATTERN_NODE_TO_EXCLUDE: True,
+        }
+    )
+    conv_node_1 = pattern.add_node(
+        **{GraphPattern.LABEL_ATTR: "CONVOLUTION", GraphPattern.METATYPE_ATTR: om.ONNXConvolutionMetatype}
+    )
```

Comment:
Could this convolution has a bias? Should we add pattern with bias as well?

Reply:
In ONNX bias is presented as an attribute of a node, e.g. Convolution. Though it could be presented as an Add node, I haven't seen such cases before. So we don't add all possible imagined combinations here

Reply:
I suggest to add the pattern with biases as it is was done in TORCH
https://github.com/openvinotoolkit/nncf/pull/2177#issue-1928297414

It could save time in future. This is not a blocking comment